src/components/Dialogs/ImageCropDialog/cropper.ts

```diff
@@ -44,6 +44,8 @@ const CANVAS_OPTS_PER_TYPE: Record<CropperImageType, Cropper.GetCroppedCanvasOpt
   },
 }
 
+export type CropBoxData = Cropper.CropBoxData
```

Comment:
Sorry for a late comment on this:

Maybe we actually elevate this to `src/types`? It will be used for some logic as well. Let's call it `ImageCropData`

Reply:
Wait did I make this comment already after all? I see it's in the `src/types` but also here 😅

Reply:
Yup, forgot to clean that up after fiddling with restoring crop feature